README.md

```diff
@@ -2,31 +2,89 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Discuss what a Driver class is in Java
 
-## Introduction
+## What is a Driver?
 
-A 1-2 sentence summary of what will be covered.
+Sometimes in Java, when we want to bring multiple classes together to
+create objects and call methods, we will create a **driver** or **runner** class.
+A driver class is often just a class with a `main()` method so that it can be
+made runnable in Java. We can import numerous classes to instantiate and execute
+various methods to "drive" or "run" the program.
 
-## Topic 1
+For example, we can define the `Liquid` class
+with various instance variables and methods, and a separate
+driver class named `Main` that contains a `main` method to instantiate
+`Liquid` instances and call the methods:
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+![multiple class folder structure](https://curriculum-content.s3.amazonaws.com/6676/java-multipleclasses/multiple_folder.png)
 
-## Topic 2
+```java
+public class Liquid
+{
+    private int currentTemp;
+    private int boilingPoint;
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+    public Liquid(int currentTemp, int boilingPoint) {
+        this.currentTemp = currentTemp;
+        this.boilingPoint = boilingPoint;
+    }
 
-## Conclusion
+    public void setCurrentTemp(int currentTemp) {
+        this.currentTemp = currentTemp;
+    }
 
-A short one or two paragraph summary of the contents of the lessons, recapping
-the learning goals.
+    public boolean isBoiling()
+    {
+        return currentTemp >= boilingPoint;
+    }
+}
+```
 
-## Resources
 
-- [Resource Link 1](example.com)
-- [Resource Link 2](example.com)
+The `Main` class is an example of a driver class.  Notice
+the `main` method must call the public methods of the
+`Liquid` class to access or modify the private instance variables. 
+
+```java
+public class Main {
+
+    public static void main(String[] args) {
+        //parameters are current temperature and boiling point
+        Liquid potOfMilk = new Liquid(198, 202);
+        Liquid potOfWater = new Liquid(230, 212);
+
+        System.out.println( potOfWater.isBoiling() );
+        System.out.println( potOfMilk.isBoiling() );
+        potOfMilk.setCurrentTemp(205);
+        System.out.println( potOfMilk.isBoiling() );
+    }
+}
+```
+
+### Multiple classes in Java Visualizer
+
+IntelliJ's debugger and Java visualizer can work with multiple
+public classes each stored in a separate file, as is the case with
+`Liquid` and `Main`.
+
+```java
+public class Liquid {
+    .....
+}
+```
+
+```java
+public class Main {
+    ....
+}
+```
+
+
+However, the `Python Tutor` visualizer requires a single public class.
```

Comment:
Just because I'm not sure if `Python Tutor` will be completely obvious to students, could we also say

"However, the `Python Tutor` or browser-based visualizer requires a single public class..." (just to clarify the web browser one)

Reply:
thanks, updated.